src/js/api/core-shim.js

```diff
@@ -314,6 +315,17 @@ function logWarning(warning) {
     console.warn(PlayerError.logMessage(warning.code));
 }
 
+function setLiveSyncDurationAttribute(model) {
```

Comment:
Can we move `getLiveSyncDuration(liveSyncDuration)` in commercial (https://github.com/jwplayer/jwplayer-commercial/pull/7304/files#diff-98c5c29fe89f7774786f064799c89076R89) to a helper in config.js here so we don't duplicate this clamping logic?

The this could look like:
```js
const liveSyncDuration = getLiveSyncDuration(model.get('liveSyncDuration'));
model.set('liveSyncDuration', liveSyncDuration);
```

But then do we have to do this here or could we just do it in config.js or playlist.js?